docs/administration.platforms.md

```diff
@@ -41,3 +74,28 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 > Currently only OpenShift is supported for Quota Management
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
+
+### Platform Deactivation
+
+Via the "Danger Zone" tab you can deactivate or reactivate your meshPlatform.
+
+Deactivation will make the meshPlatform not available anymore to new meshTenants, existing meshTenants won't be accessible via meshPanel anymore and no more replication will be executed. Metering instead will still be executed for non-deleted meshTenants of this meshPlatform for now. This will change in future. Direct access in the cloud platform itself to the platform tenants is still possible as meshStack does not delete or do any updates on platform tenants when a meshPlatform is deactivated.
```

Comment:
"This will change in future". This sounds a bit vague, can we by any chance already provide more info on the topic or even get rid of this sentence altogether?

Reply:
I'm in favor of getting rid of it and just adapt the description when the behavior is changed,